README.md

```diff
@@ -40,6 +40,9 @@ To try out this example app:
    $ west build zephyr-esp32-example
    ```
 
+   To build the MCUboot image as well, which will also get flashed in the
+   following step, pass `--sysbuild`.
```

Comment:
Will the board boot without this / can we make it the default? The NCS apps seem to enable it by default but they may be having it in, not sure

Reply:
It boots as long as MCUboot has been flashed before. I want to check whether it will boot w/ the esp32 bootloader but zephyr doesn't have support for the c3-devkitc or c6, just the c3-devkitm and my s3-devkitm already has mcuboot :(  but I think it is safe to assume it doesn't boot, so you're right that we should enable the sysbuild by default

> The NCS apps seem to enable it by default

Dug a bit into this, and learned that the [concept of a child image](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/config_and_build/multi_image.html#defining-and-enabling-a-child-image) we see in NCS samples is an NCS concept -- and is mutually exclusive with sysbuild. They actually [shut off creating a child image](https://github.com/nrfconnect/sdk-nrf/blob/014d2093aa61f9e9e293417dc58b73698f615453/cmake/multi_image.cmake#L7-L14) if sysbuild is enabled. Pretty neat. We could have sysbuild enabled by default with the config command [`west config build.sysbuild True`](https://docs.zephyrproject.org/latest/build/sysbuild/index.html#building-with-sysbuild). Added that config and it works. I wish we could builtin like NCS but I think this solves the concern of them forgetting to pass `--sysbuild`?

Reply:
Dang, I can't see how to force it by default without users needing to run `west config --local...`, that's too bad. We can document it at least.

Reply:
You can use esptool.py to erase flash, in case you need it

Reply:
Ah yea we'd want `--local` not global, added!